src/js/lib/yasr.bundled.js

```diff
@@ -56918,7 +56926,7 @@ var root = module.exports = function(yasr) {
 		var contentType = yasr.results.getOriginalContentType();
 		var type = yasr.results.getType();
 		return {
-			getContent: function() {return yasr.results.getOriginalResponseAsString();},
+			getContent: function() {return yasr.results.getOriginalResponse();},
```

Comment:
@sava-savov-ontotext Where this change came from? Any idea?

Reply:
Probably this is a kind of a fix that @MartinaDimova has done but which was not included into YASR project but fixed straight into WB project

Reply:
Well, this is not a good practice at all